### PR TITLE
kubernetes blog post

### PR DESCRIPTION
**Pull request by RPrudden**

### CODE REVIEW

File: _posts/2016-03-30-kubernetes-in-pics.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+layout:     post
+title:      "Kubernetes in pictures"
+date:       2016-03-30
+summary:    A symposium on interactive documentary.
+categories: ['technical','infrastructure']
+author: 	Rachel Prudden
+project:    none
+thumbnail:  "https://s3-eu-west-1.amazonaws.com/informatics-webimages/articles/2016-03-30-kubernetes-in-pics/cluster.jpg"
+header: "https://s3-eu-west-1.amazonaws.com/informatics-webimages/vr.jpg"
+---
```
Comment (jacobtomlinson) [inline _posts/2016-03-30-kubernetes-in-pics.md]: You might want to check some of this meta data. I think the summary and header images are wrong.

Comment (RPrudden) [inline _posts/2016-03-30-kubernetes-in-pics.md]: ha, thanks. I meant to reuse the header, but not the summary!

Comment (RPrudden) [inline _posts/2016-03-30-kubernetes-in-pics.md]: Fixed

**Comment by jacobtomlinson** [comment]
Great! I like it. Concise and straightforward.

I particularly like the inclusion of the pens in the photos!

Shout when you're happy and I'll merge.

**Comment by RPrudden** [comment]
Thanks! I'm happy now.

Ross can claim credit for the pens :)

**Pull request merged**